src/Framework/FieldsAPI/Concerns/HasPersistence.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Give\Framework\FieldsAPI\Concerns;
+
+use Give\Framework\FieldsAPI\ValueObjects\PersistenceScope;
+
+/**
+ * This provides the ability to set a scope and meta key for a field. The scope is used to determine if and where the
+ * field should be stored. The meta key is used to store the field in the database.
+ *
+ * @unreleased
+ */
+trait HasPersistence
+{
+    /**
+     * @unreleased
+     *
+     * @var PersistenceScope|null
+     */
+    protected $scope;
+
+    /**
+     * @unreleased
+     *
+     * @var string|null
+     */
+    protected $metaKey = null;
+
+    /**
+     * @unreleased
+     *
+     * @param string|PersistenceScope $scope
+     */
+    public function scope($scope): self
```

Comment:
Yeah, I thought about that, too. I think I'm learning towards `scope` as it's simple and I can't think of another concept it would clash with. I know I'm talking about persistence elsewhere, but to say the "scope" of a field is "fee recovery" feels sensible to me.